quickwit-cli/src/main.rs

```diff
@@ -32,12 +32,17 @@ use quickwit_metastore::IndexMetadata;
 use quickwit_metastore::MetastoreUriResolver;
 use quickwit_proto::SearchRequest;
 use quickwit_search::single_node_search;
+use quickwit_serve::serve_cli;
```

Comment:
Maybe we can refactor by following what we have been doing. create `serve_cli` in the `quickwit-cli` crate to allow it to display user messages like.:
```bash 
Server running on http://localhost:8000  
You can search with `curl http://localhost:8000/api/v1/search?query=ada`
```